opm-test.sh

```diff
@@ -1,26 +1,58 @@
 #!/bin/bash
-set -euxo pipefail
 
-echo "Setting Variables"
-read -p "Bunlde image: " bundleImage
-read -p "Index image tag: " indexImage
-read -p "docker or podman: " containerTool
+username=$2
+token=$3
+echo "logging in to quay"
+docker login -u "$username" -p "$token" quay.io
```

Comment:
Shouldn't this support docker or podman?

Reply:
podman shares that auth